extensions/debuggee/src/org/exist/debugger/client/DebugClient.java

```diff
@@ -40,28 +41,29 @@
     private final static String COMMAND_STEP = "step";
     private final static String COMMAND_CONT = "cont";
 
-    private ConsoleReader console;
+    //TODO:check where that class would be
+    //private ConsoleReader console;
     private Debugger debugger;
     private DebuggingSource source = null;
 
     public DebugClient() throws IOException {
-        Terminal.setupTerminal();
-        console = new ConsoleReader();
+//        Terminal.setupTerminal();
+//        console = new ConsoleReader();
         debugger = DebuggerImpl.getDebugger();
     }
 
     public void readlineInputLoop() {
         String prompt = "offline";
         boolean cont = true;
         while (cont) {
-            try {
-                String line = console.readLine(prompt + "> ");
-                cont = parseInput(line);
-            } catch (EOFException e) {
-                break;
-            } catch (IOException e) {
-                System.err.println("Exception caught: " + e.getMessage());
-            }
+//            try {
+//                String line = console.readLine(prompt + "> ");
+//                cont = parseInput(line);
+//            } catch (EOFException e) {
+//                break;
+//            } catch (IOException e) {
+//                System.err.println("Exception caught: " + e.getMessage());
+//            }
```

Comment:
As per previous comment...

Reply:
Follow up problem based on the missing import